sdk/tests/deqp/functional/gles3/es3fTransformFeedbackTests.js

```diff
@@ -978,323 +1005,296 @@ goog.scope(function() {
-            // debug(genString(buffer, "output data"));
-
-            // Verify all output variables that are written to this buffer.
-            for (var i = 0; i < this.m_transformFeedbackOutputs.length; ++i) {
-                var out = this.m_transformFeedbackOutputs[i];
-
-                if (out.bufferNdx != bufferNdx)
-                    continue;
-
-                var inputOffset = 0;
-                var outputOffset = 0;
-
-                // Process all draw calls and check ones with transform feedback enabled
-                for (var callNdx = 0; callNdx < calls.length; ++callNdx) {
-                    var call = calls[callNdx];
-
-                    if (call.transformFeedbackEnabled) {
-                        var inputPtr = inputData[0] + inputOffset * this.m_inputStride; // const deUint8*
-                        var outputPtr = outputOffset * stride; // const deUint8*
-
-                        if (!es3fTransformFeedbackTests.compareTransformFeedbackOutput(this.m_primitiveType, out, call.numElements, {
-                                 input: {
-                                    buffer: inputData,
-                                    offset: inputOffset * this.m_inputStride,
-                                    stride: this.m_inputStride
-                                },
-                                output: {
-                                    buffer: buffer,
-                                    offset: outputOffset * stride,
-                                    stride: stride
-                                }
-                            })) {
-                            outputsOk = false;
-                            break;
-                        }
-                    }
-
-                    inputOffset += call.numElements;
-                    outputOffset += call.transformFeedbackEnabled ? es3fTransformFeedbackTests.getTransformFeedbackOutputCount(this.m_primitiveType, call.numElements) : 0;
-
-                }
-            }
-
-            // Verify guardband.
-            if (!es3fTransformFeedbackTests.verifyGuard(buffer, size)) {
-                bufferedLogToConsole('Error: Transform feedback buffer overrun detected');
-                outputsOk = false;
-            }
-
-        //    Javascript, and lazy memory management
-        //    gl.unmapBuffer(gl.TRANSFORM_FEEDBACK_BUFFER);
-
-        }
-
-        // Check status after mapping buffers.
-        var mustBeReady = this.m_outputBuffers.length > 0; // Mapping buffer forces synchronization. // const bool
-        var expectedCount = es3fTransformFeedbackTests.computeTransformFeedbackPrimitiveCount(this.m_primitiveType, calls); // const int
-        var available = false; // deUint32
-        var numPrimitives = 0; // deUint32
+    	// Compare result buffers.
+    	for (var bufferNdx = 0; bufferNdx < this.m_outputBuffers.length; ++bufferNdx) {
+    		var stride = this.m_bufferStrides[bufferNdx]; // int
```

Comment:
8-space indentation looks aggressive here. Could we shorten it to 4 similar to elsewhere?